enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk_setup/components/SdkIframeEmbedSetupProvider.tsx

```diff
@@ -31,32 +34,36 @@ export const SdkIframeEmbedSetupProvider = ({
 }: SdkIframeEmbedSetupProviderProps) => {
   const [isEmbedSettingsLoaded, setEmbedSettingsLoaded] = useState(false);
 
+  const [rawSettings, setSettings] = useState<SdkIframeEmbedSetupSettings>();
+
+  const [persistedSettings, persistSetting] = useUserSetting(
```

Comment:
```suggestion
  const [persistedSettings, persistSettings] = useUserSetting(
```

So the names match

Reply:
Fixed in https://github.com/metabase/metabase/pull/60234/commits/932827d82adfde4289a1550575a5a03855ad4e5f

Reply:
Fixed in https://github.com/metabase/metabase/pull/60234/commits/0b74bdfad351ad1bcf20ee58512ffdaf916f5a7a